src/shared/conf-parser.h

```diff
@@ -100,6 +100,7 @@ int config_parse_many_nulstr(
                 void *userdata);
 
 int config_parse_many(
+                char ***ret_dropins,        /* possibly NULL */
                 const char *conf_file,      /* possibly NULL */
                 const char* const* conf_file_dirs,
                 const char *dropin_dirname,
```

Comment:
Looks nice, except for one nitpick: I'd prefer to have the output parameter last. I know this is a bit of bike-shedding, and if you disagree I'm fine with leaving this form, but in general the coding style is to have output parameters last. The only exception is "constructor" functions, which doesn't apply to this case.

Reply:
OK. Done.